lib/resources/ConferenceLocation.ts

```diff
@@ -0,0 +1,73 @@
+import * as Promise from 'bluebird';
+
+import {
+    HudAiCreateAttributes,
+    HudAiListAttributes,
+    HudAiUpdateAttributes,
+    Resource
+} from '../utils/Resource';
+import { RequestManager } from '../RequestManager';
+import {ConferenceLocation} from '../entities/ConferenceLocation';
+
+
+
+export interface ConferenceLocationListAttributes extends HudAiListAttributes {
+    name?: string;
+    city?: string;
+    state?: string;
+    zip?: string;
+    country?: string;
+    conferenceId?: string;
+}
+
+export interface ConferenceLocationCreateAttributes extends HudAiCreateAttributes {
+    name?: string;
+    line1: string;
+    line2?: string | null;
+    city: string;
+    state: string;
+    zip: string;
+    country: string;
+    conferenceId: string;
+}
+
+export interface ConferenceLocationUpdateAttributes extends HudAiUpdateAttributes {
+    name?: string;
+    line1?: string;
+    line2?: string | null;
+    city?: string;
+    state?: string;
+    zip?: string;
+    country?: string;
+    conferenceId?: string;
```

Comment:
I would imagine that the use case here will almost always be updating location based on conference id

Reply:
You still want this endpoint available regardless. My current use case actually doesn't require update by conferenceId but we can add it when the need arises. Returning locations with conferences would be an api change